Update default.html

ascyrax opened the pull request:
Previous youtube channel link was dead.

ascyrax committed:
Update default.html

Previous youtube channel link was dead.

KevinWorkman commented:
Nice catch! I must have accidentally changed this when I changed my Twitter profile. Thanks!

ascyrax commented:
> 



> Nice catch! I must have accidentally changed this when I changed my Twitter profile. Thanks!



> Nice catch! I must have accidentally changed this when I changed my Twitter profile. Thanks!

Most welcome🙂

ascyrax commented:
> Nice catch! I must have accidentally changed this when I changed my Twitter profile. Thanks!

Most welcome 🙂
